.github/workflows/slash-commands.yml

```diff
@@ -8,17 +8,29 @@ jobs:
     if: ${{ github.event.issue.pull_request }}
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+
+      - name: Check PAT rate limits
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.AIRBYTEIO_PAT }} \
+            ${{ secrets.OSS_BUILD_RUNNER_GITHUB_PAT }} \
+            ${{ secrets.SUPERTOPHER_PAT }} \
+            ${{ secrets.DAVINCHIA_PAT }}
+
       - name: Get PR repo and ref
         id: getref
         run: |
           pr_info="$(curl ${{ github.event.issue.pull_request.url }})"
           echo ::set-output name=ref::"$(echo $pr_info | jq -r '.head.ref')"
           echo ::set-output name=repo::"$(echo $pr_info | jq -r '.head.repo.full_name')"
+
       - name: Slash Command Dispatch
         id: scd
         uses: peter-evans/slash-command-dispatch@v2
         with:
-          token: ${{ secrets.SUPERTOPHER_PAT }}
+          token: ${{ env.PAT }}
```

Comment:
this is a real solid move @zzztimbo 

are all `env` vars hidden by default?

![](https://media2.giphy.com/media/TL5XQpSpASo4U/giphy.gif?cid=ecf05e47311qw59t92o6hdrvlgrg0y2rpn7r73lifwdf2ikd&rid=giphy.gif&ct=g)

Reply:
Nope. Github recognizes the PAT as a secret, but probably not other strings?
![image](https://user-images.githubusercontent.com/4597/188685738-a9a5065b-c20b-474c-8992-4b064f609f18.png)